implementers-guide/sections/02-introduction.adoc

```diff
@@ -0,0 +1,11 @@
+
+
+== Introduction
+
+This implementers' guide is a compilation of reported defects for all versions of the {Rec # | Series} of Recommendations. In this edition of the guide, reported defects identified as of {yyyy-mm} are given for:
+
+
+// list of recommendations to which the text applies
```

Comment:
Can we make this into actual text so it renders for the user?

Reply:
Yes, change applied.